lib/v2/huoxian/zone.js

```diff
@@ -0,0 +1,17 @@
+const got = require('@/utils/got');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const { data } = await got(`https://zone.huoxian.cn/api/discussions?sort=-createdAt`);
+    const items = data.data.map((item) => ({
+        title: item.attributes.title,
+        link: `https://zone.huoxian.cn/d/${item.attributes.slug}`,
+        description: item.attributes.title,
+        pubDate: parseDate(item.attributes.createdAt),
```

Comment:
```suggestion
        description: data.included.find((i) => i.id === item.relationships.firstPost.data.id).attributes.contentHtml,
        pubDate: parseDate(item.attributes.createdAt),
        author: data.included.find((i) => i.id === item.relationships.user.data.id).attributes.displayName,
        category: data.included.filter((i) => item.relationships.tags.data.map((t) => t.id).includes(i.id) && i.type === 'tags').map((i) => i.attributes.name),
```
The full article, author, and tags are also available from the API. Please make good use of them.